testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-azure/update-location.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+/usr/bin/mysqld_safe &
+while ! mysqladmin ping -proot --silent; do sleep 1; done
+
+hive --service metatool -updateLocation "abfs://${ABFS_CONTAINER}@${ABFS_ACCOUNT}.dfs.core.windows.net/${ABFS_SCHEMA}" hdfs://hadoop-master:9000/user/hive/warehouse
```

Comment:
i'd argue that we could take care of these aspects in a subsequent PR.

creating external tables fits to the tested scenario - no need to change it as part of this PR

Reply:
This is required for adding `fs.hadoop.enabled=false` in `hive.properties`.